policyengine_uk/variables/gov/dwp/state_pension.py

```diff
@@ -25,6 +25,25 @@ def formula(person, period, parameters):
         return age >= threshold
 
 
+class reached_SPA_before_2016(Variable):
```

Comment:
```suggestion
class meets_savings_credit_age_requirement(Variable):
```

and make `2016` a policy parameter